docs/developing/autofocus.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Autofocus
```

Comment:
I think we should call this "Managing Focus" or something similar. Much of the content here has to do with `setFocus`, though `autofocus` is one instance where `setFocus` comes into play.